Formula/geocode-glib.rb

```diff
@@ -35,10 +34,6 @@ def install
     end
   end
 
-  def post_install
-    system Formula["gtk+3"].opt_bin/"gtk3-update-icon-cache", "-f", "-t", "#{HOMEBREW_PREFIX}/share/icons/gnome"
```

Comment:
Is this no longer necessary? Why not create `HOMEBREW_PREFIX/"share/icons/gnome"` if there are complaints about it not existing?